src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+}
+
+.product {
```

Comment:
The BEM block should be named the same as file. 
Rename the product into card to match the task description

Reply:
Done